taxi/views.py

```diff
@@ -87,3 +90,36 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreationForm
+    success_url = "taxi:driver-list"
```

Comment:
The `success_url` should be wrapped with `reverse_lazy` to ensure it resolves correctly. Change `success_url = "taxi:driver-list"` to `success_url = reverse_lazy("taxi:driver-list")`.